site/src/components/Dashboard/DashboardProvider.tsx

```diff
@@ -129,3 +129,8 @@ export const useTemplatePoliciesEnabled = (): boolean => {
     experiments.includes("template_update_policies")
   );
 };
+
+export const useAccessControlEntitled = (): boolean => {
+  const { entitlements } = useDashboard();
+  return entitlements.features.access_control.enabled;
+};
```

Comment:
I'm ok with this but since it is only two lines of code do you think it needs to be a custom hook?

Reply:
Probably not? I thought it was a pattern, was just copying above. Removed :+1: